.github/workflows/hb_library_build_workflow.yml

```diff
@@ -0,0 +1,72 @@
+name: hb_wheel
+
+on:
+  push:
+    branches: [master, development]
+
+jobs:
+  build_hummingbot:
+    name: build hb wheel
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      # Use cache's hashFiles function to check for changes in core code
+      - name: Check for code changes
+        id: program-changes
+        uses: actions/cache@v2
+        env:
+          # Increase this value to manually reset cache if program files have not changed
+          CACHE_NUMBER: 0
+        with:
+          path: README.md # placeholder file
+          key: ${{ runner.os }}-build-${{ env.CACHE_NUMBER }}-${{ hashFiles('hummingbot/*', '**/*.py', '**/*.py*', '**/*.pxd', 'test/*') }}
+
+      # Check for setup/environmnet-linux.yml changes
+      - name: Cache conda dependencies
+        id: conda-dependencies
+        uses: actions/cache@v2
+        env:
+          # Increase this value to manually reset cache if setup/environment-linux.yml has not changed
+          CONDA_CACHE_NUMBER: 0
+        with:
+          path: |
+            /home/runner/conda_pkgs_dir/
+            /usr/share/miniconda/envs
+          key: ${{ runner.os }}-conda-${{ env.CONDA_CACHE_NUMBER }}-${{ hashFiles('setup/environment-linux.yml') }}
+
+      # Install environment and Hummingbot
+      - name: Install environment and Hummingbot
+        uses: ./.github/actions/install_env_and_hb
+        with:
+          program-cache-hit: ${{steps.program-changes.outputs.cache-hit}}
+          dependencies-cache-hit: ${{steps.conda-dependencies.outputs.cache-hit}}
+
+      # Build wheel file
+      - name: Build library wheel
+        shell: bash
+        if: steps.program-changes.outputs.cache-hit != 'true' || steps.conda-dependencies.outputs.cache-hit != 'true'
+        run: |
+          source $CONDA/etc/profile.d/conda.sh
+          conda activate hummingbot
+          rm -f library/hummingbot*linux.whl
+          python3 setup.py bdist_wheel
+          mv dist/hummingbot*linux*.whl library/
+      # Commit and push generated wheel file
+      - name: Commit wheel file
+        shell: bash
+        if: steps.program-changes.outputs.cache-hit != 'true' || steps.conda-dependencies.outputs.cache-hit != 'true'
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git add -f library/*
+          git commit -m "Build Hummingbot library wheel"
```

Comment:
Maybe we should indicate here and in `hb_library_build_workflow_mac.yml` that one is a commit for Linux wheel and the other is for MacOS

Reply:
Changed